### PR TITLE
Added paparazzi screen test for 5 more module

### PR DESCRIPTION
**Pull request by Vivekban**
## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [ ] The PR is submitted to the `main` branch.
- [] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) and my PR doesn't break the rules.
- [] I've read and understand the [Developer Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Guidelines.md).
- [ ] I confirm that I've run the code locally and everything works as expected.
- [ ] 🎬 I've attached a **screen recoding** of the changes. 

> Tip: drag & drop the video to the PR description.

## What's changed?
Added paparazzi screenshots for Account, Attribution, Balance, Budgets and contributions modules.

Before | After
--|--
image1 | image2
-->

Describe with a few bullets **what's new:**
- Added paparazzi screenshots for Account, Attribution, Balance, Budgets and contributions modules.
 

> 💡 Tip: Please, attach screenshots and screen recordings. It helps a lot!

## Risk Factors

**What may go wrong if we merge your PR?**

- Nothing

**In what cases your code won't work?**

- It will work

## Does this PR closes any GitHub Issues?

Check **[Ivy Wallet Issues](https://github.com/Ivy-Apps/ivy-wallet/issues)**.

- Closes #ISSUE_NUMBER

> Replace `ISSUE_NUMBER` with the number of the GitHub issue that this PR is fixing. If you've done that correctly, you'll see the issue title linked when previewing your PR description.

> **!Note:** Do **not** link the PR number. Link the number/id of the GitHub Issue from [issues](https://github.com/Ivy-Apps/ivy-wallet/issues).

## Troubleshooting CI failures ❌

GitHub Actions failing? Read our [CI Troubleshooting guide](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/CI-Troubleshooting.md).

### CODE REVIEW

**Review by ILIYANGERMANOV — APPROVED** [review]
Hey @Vivekban nice work! Can you also please a section about Paparazzi in "docs/CI-Troubleshooting.md" so folks will know what to do when Paparazzi fail. Give instructions about what are screenshot tests, `recordPaparazziDebug`, `verifyPaparazziDebug` and how to use them 🙏

**Comment by Vivekban** [comment]
@ILIYANGERMANOV Locally things are working fine for paparazzi tests as you can see in the screenshots. If possible can you take a look on your side?

image

**Comment by ILIYANGERMANOV** [comment]
Hey @Vivekban try:
```
./gradlew clean
git lfs push --all origin your-branch-name
git push
./gradlew verPaD
```

**Comment by ILIYANGERMANOV** [comment]
2 tests completed, 2 failed
> Task :screen:categories:kspDebugUnitTestKotlin

> Task :screen:budgets:testDebugUnitTest FAILED

com.ivy.budgets.BudgetScreenPaparazziTest > snapshot Balance Screen[Dark] FAILED
    java.lang.AssertionError at BudgetScreenPaparazziTest.kt:17


Run locally:
```
./gradlew screen:budgets:recPaD
```

**Comment by Vivekban** [comment]
@ILIYANGERMANOV  Same thing, not sure. Have you tried on your side?

image

**Comment by ILIYANGERMANOV** [comment]
> @ILIYANGERMANOV Same thing, not sure. Have you tried on your side?
> 
> image

Nope, I don't have time to work on Ivy these days. I bet it's something git lfs related. You see the GitHub Actions workflow in ".github/workflows"

**Comment by ILIYANGERMANOV** [comment]
@Vivekban I guess some of your paparazzi tests are flaky because they use real dates/times. All datetime logic should be hard-coded. My assumption is:
- It's some date/time issue.
- Works on your machine because you're in time zone X 
- CI is in time zone Y => it fails. 

In a nutshell, in the PR I saw some flaky paparazzi tests that deal with time.now() and system's timezone. Ideally, these should be done in the VM and the UI should display some already formatted date-time string. If it's not easy to fix just remove the paparazzi tests for the flaky screens

File: shared/domain/src/test/java/com/ivy/domain/usecase/csv/ExportCsvUseCasePropertyTest.kt [hunk 1]
```diff
@@ -44,43 +28,43 @@ class ExportCsvUseCasePropertyTest {
...
-                val hasExpectedNumOfColumns = row.size == IvyCsvRow.Columns.size
-                if (!hasExpectedNumOfColumns) {
-                    println("(${row.size} cols) $row")
-                }
-                hasExpectedNumOfColumns shouldBe true
-            }
-        }
-    }
+//    @Test
+//    fun `property - num of row and columns matches the format`() = runTest {
```
Comment (ILIYANGERMANOV) [inline shared/domain/src/test/java/com/ivy/domain/usecase/csv/ExportCsvUseCasePropertyTest.kt:32]: What? Why did you comment out this test? It's critical!

**Review by ILIYANGERMANOV — CHANGES_REQUESTED** [review]
It should not disable existing unit tests!

File: shared/domain/src/test/java/com/ivy/domain/usecase/csv/ExportCsvUseCasePropertyTest.kt (hunk 1 above)
Comment (Vivekban) [reply shared/domain/src/test/java/com/ivy/domain/usecase/csv/ExportCsvUseCasePropertyTest.kt:32]: Oh sorry, My mistake

File: screen/home/src/main/java/com/ivy/home/customerjourney/CustomerJourney.kt [hunk 2]
```diff
@@ -41,24 +44,28 @@ fun CustomerJourney(
...
+                onDismiss = {
+                    onDismiss(card)
+                }
+            ) {
+                card.onAction(nav, ivyContext, rootScreen)
             }
-        ) {
-            card.onAction(nav, ivyContext, rootScreen)
         }
+    } else {
```
Comment (ILIYANGERMANOV) [inline screen/home/src/main/java/com/ivy/home/customerjourney/CustomerJourney.kt:67]: Add a comment that this is done because of Paparazzi

**Review by ILIYANGERMANOV — APPROVED** [review]
Nice work 💯 It would be nice if we can also mock some transaction cards on the Home screen snapshot

**Pull request merged**